bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBigtableDataClient.java

```diff
@@ -94,4 +100,65 @@ Row readModifyWriteRow(ReadModifyWriteRow readModifyWriteRow)
    */
   Boolean checkAndMutateRow(ConditionalRowMutation conditionalRowMutation)
       throws ExecutionException, InterruptedException;
+
+  /**
+   * Sample row keys from a table.
+   *
+   * @param tableId a String object.
+   * @return an immutable {@link List} object.
+   */
+  List<KeyOffset> sampleRowKeys(String tableId);
+
+  /**
+   * Sample row keys from a table, returning a Future that will complete when the sampling has
```

Comment:
line 113: Sample row keys from a table, returning a Future that will complete when the sampling has completed. 
line 133: @return a {@link ListenableFuture} that will finish when
   * all reads have completed.
line 158: @return a {@link com.google.common.util.concurrent.ListenableFuture} that will finish when all reads have completed.

Looking at all these lines, I would suggest the following: 
line 113: Sample row keys from a table; returns a Future that ends when the sampling has completed. 

line 133: @return a {@link ListenableFuture} that ends when
   * all reads have completed.

line 158: @return a {@link com.google.common.util.concurrent.ListenableFuture} that ends when all reads have completed.